lib/cc/cli/init.rb

```diff
@@ -64,9 +64,8 @@ def create_default_engine_configs
       end
 
       def available_engine_configs
-        engine_names = existing_cc_config.engines.select do |_, config|
-          config.enabled?
-        end.keys
+        engines = existing_cc_config.engines || {}
```

Comment:
This was discussed a bit on [#312](https://github.com/codeclimate/codeclimate/pull/312#discussion_r50721192) when this was implemented. `existing_cc_config` is guaranteed to exist when this is called: either it will have been generated by this class just now, or it already exists (and was upgraded from classic, if it needed to be).

The crash that did occur was that `engines` is `nil` instead of `{}` in `CC::Yaml` when it's empty. Maybe that should be fixed instead.